utils/http-calls.js

```diff
@@ -0,0 +1,11 @@
+import { SERVER_URL } from '../config';
+
+export async function apiFetch(endpoint) {
+    let response = await fetch(`${SERVER_URL}/api/${endpoint}`);
+
+    if (response.status >= 400 && response.status < 600) {
```

Comment:
There are libraries that contains those error codes and their name, use that instead of writing 400 and 600

Reply:
Used response.ok